### PR TITLE
Landing page Update

### PR DESCRIPTION
**Pull request by Brayve-Pancake**
Hi Bitbeckeres,

CHANGES
1. The updated landing page.
The landing page is self-contained and does not interact with other URLs.
2. The NavBar.
This component is used across the site.

NEW STUFF
Only one new package has been installed: SwiperJS - for the carousel.

YARN
Please note, this is the first time I've used Yarn instead of NPM. I don't expect this to pose any issues as I could get the app to run locally. Please check the Yarn Lock. I've shared the updates here so you can determine if they should be included or excluded.

### CODE REVIEW

File: contracts/.DS_Store
Comment (bitbeckers) [inline contracts/.DS_Store:1]: This one can go ;)

File: .DS_Store
Comment (bitbeckers) [inline .DS_Store:1]: This one can go as well

Comment (Brayve-Pancake) [reply .DS_Store:1]: Do I need to do anything here, I thought that the privileges allow you to remove this?

If you're just letting me know what's happening, then great :)

Comment (bitbeckers) [reply .DS_Store:1]: I've removed them now, but they shouldn't have been in the commit in the first place

**Pull request merged**